src/fontra/workflow/actions.py

```diff
@@ -224,6 +225,10 @@ def _scaleStaticGlyph(
             if glyph.verticalOrigin
             else glyph.verticalOrigin
         )
+        anchors = [
+            Anchor(name=a.name, x=a.x * self.scaleFactor, y=a.y * self.scaleFactor)
```

Comment:
This would drop any customData. The proper pattern is to use `replace()`, from the dataclasses module, like this:
```python
        anchors = [
            replace(a, x=a.x * self.scaleFactor, y=a.y * self.scaleFactor)
            for a in glyph.anchors
        ]
```
Which is also shorter as we don't need to set the name attr.

Reply:
Nice. Thanks a lot for the hint! Done.